lib/this_feature/adapters/memory.rb

```diff
@@ -34,6 +34,28 @@ def control?(flag_name, **kwargs)
         !present?(flag_name)
       end
 
+      # def treatment_config(flag_name, context: nil, data: {}, record: nil)
```

Comment:
?

Reply:
planning to continue with this method after I got some feedback on the rest of the implementation so far